Documentation/ApiOverview/ContentElements/CustomBackendPreview.rst

```diff
@@ -14,6 +14,72 @@ following sections describe how to achieve that.
 A preview renderer is used to facilitate (record) previews in TYPO3. This
 class is responsible for generating the preview and the wrapping.
 
+The default preview renderer is :php:`StandardContentPreviewRenderer` and handles
+the Core's built-in content types (field :sql:`CType` in table :sql:`tt_content`).
+
+Extend the default PreviewRenderer
+==================================
+
+There are two ways of how to provide previews for your own custom content types:
+via page TSconfig or event listener.
+
+page TSconfig
```

Comment:
```suggestion
Page TSconfig
```

Start of the heading, capital P.

Additionally, what is the advantage using this variant? As a reader I want to have a quick overview what's best to use in my concrete situation.